static/scss/certificates/certificate.scss

```diff
@@ -292,11 +437,34 @@
     font-weight: bold;
     font-size: 16px;
     line-height: 22px;
-    color: #333;
+    color: $cert-img-holder-color;
   }
 }
 
 @media print {
+  a {
+    color: black !important;
+  }
+
```

Comment:
Kindly add the following to restrict `help` button from getting printed:
```
#launcher {
    display: none;
}
```

Reply:
Updated